cmd/release-controller/info.go

```diff
@@ -166,6 +179,47 @@ func (r *ExecReleaseInfo) ChangeLog(from, to string) (string, error) {
 	return out.String(), nil
 }
 
+func (r *ExecReleaseInfo) Bugs(from, to string) (string, error) {
```

Comment:
this should return a struct and you'll need to define a groupcache.Sink for it.  We would anticipate bringing additional info back so let's lay the foundation now (the struct is a good minimal piece)

Reply:
We can't make a `groupcache.Sink` for our own structs. One of the functions for the interface is intentionally not exported to prevent us from making custom `Sink`s: https://github.com/golang/groupcache/issues/89

One alternative is marshalling and unmarshalling to and from JSON (or any other encoding that works for us).

Reply:
Ok, I think it's fine for now to leave this as is.  Document on the godoc for the method what string contains (newline delimited bug numbers).